src/Microsoft.Azure.WebJobs.Host/Loggers/Logger/Constants/ScopeKeys.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+namespace Microsoft.Azure.WebJobs.Host.Loggers
+{
+    internal static class ScopeKeys
+    {
+        // These are used internally for passing values via scopes
+        public const string FunctionInvocationId = "MS_FunctionInvocationId";
+        public const string FunctionName = "MS_FunctionName";
+        public const string HttpRequest = "MS_HttpRequest";
+
+        // HTTP context is set automatically by ASP.NET, this isn't ours.
+        internal const string HttpContext = "MS_HttpContext";
+
+        // This is set by Functions
+        internal const string FunctionsHttpResponse = "MS_AzureFunctionsHttpResponse";
```

Comment:
Thus far, we've had the flexibility of renaming this key, since it is currently only used internally in Functions. If we do rename it in the future, how can we have a test that will catch that?